store/store.go

```diff
@@ -378,3 +378,28 @@ func (s *store) WriteBatch() error {
 func (s *store) IsBanned(addr crypto.Address) bool {
 	return s.config.BannedAddrs[addr]
 }
+
+func (s *store) pruneBlock(batch *leveldb.Batch, blockHeight uint32) (bool, error) { //nolint
```

Comment:
why `//nolint`?

Reply:
It's a private function and currently its not used. once it used by other functions we can remove it.